.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+<!--
+Thanks for submitting a change to gherlint!
+
+To make it possible for us to get your change reviewed and merged please carefully fill out the requested information below.
+
+Please note that any kind of change needs first be submitted to the master branch which holds the next version of gherlint.
+
+Please set the following labels:
+
+- Set label "Status:Needs-Review" for review or "Status:In-Progress" if the PR still has open tasks.
+- Assignment: assign to self
+- Reviewers: pick at least one
+-->
+
+## Description
+
```

Comment:
```suggestion
<!--- a good and descriptive information about the proposed changes -->
```

Reply:
Suggested not to keep these kinds of comments as we have contributing guidelines.
@saw-jan @kiranparajuli589